scripts/pyro_camera_server.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""
+Script to run a Pyro camera server. This should be run on the control computer for a distributed
+camera. The configuration for the camera is read from the pyro_camera.yaml/pyro_camera_local.yaml
+config file. The camera servers should be started after the name server, but before POCS.
```

Comment:
This strikes me as problematic. It would seem to be best to have a rendezvous that can allow for a uncoordinated starting, and especially restarting (e.g. one camera is wonky, so you power cycle it without requiring POCS to be shutdown).